recipe/meta.yaml

```diff
@@ -47,7 +47,7 @@ test:
   commands:
     - ncks --help
     - ncap2 --help
-    - ncks -M "http://tds.marine.rutgers.edu/thredds/dodsC/roms/espresso/2013_da/his/ESPRESSO_Real-Time_v2_History_Best"  # [not osx]
+    - ncks -M "http://tds.marine.rutgers.edu/thredds/dodsC/roms/espresso/2013_da/his/ESPRESSO_Real-Time_v2_History_Best"
```

Comment:
A colleague of mine tested this on his Mac and it worked fine as far as we could tell, something like:
```
netcdf ESPRESSO_Real-Time_v2_History_Best {
  dimensions:
    boundary = 4 ;
    eta_psi = 81 ;
    eta_rho = 82 ;
    eta_u = 82 ;
    eta_v = 81 ;
    ocean_time = UNLIMITED ; // (24 currently)
    s_rho = 36 ;
    s_w = 37 ;
    time = 45516 ;
    tracer = 2 ;
    xi_psi = 129 ;
    xi_rho = 130 ;
    xi_u = 129 ;
    xi_v = 130 ;
...
    :DODS_EXTRA.Unlimited_Dimension = "ocean_time" ;
    :EXTRA_DIMENSION.N = 36 ;
} // group /
```